decidim-core/spec/cells/decidim/diff_cell_spec.rb

```diff
@@ -18,7 +18,7 @@
 
   context "when diffing a translatable attribute that has empty strings" do
     let(:title) { { en: "English title", ca: "Catalan title", es: "" } }
-    let(:item) { create(:result, title: title) }
+    let(:item) { create(:initiative, title: title) }
```

Comment:
We should extract an example and include it in a spec in initiatives please

Reply:
I'm removing both changes above. They were mostly me trying to 'fix' a red in CI coverage tasks that was not really because of the changes.